### PR TITLE
Update readme: getting started

### PR DESCRIPTION
**Pull request by richardkapolnai-da**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,75 +11,63 @@ Issuing a new bond is currently a fragmented process. A Distributed Ledger Techn
...
 
-#### Stopping Stand-Alone Run
-1.  Stop the bots by pressing **Ctrl+C**.
-2.  Stop the Sandbox: type:
-    ```
-    daml stop
-    ```
+#### Stopping Standalone Run
+1. Stop the bots by pressing **Ctrl+C**.
+1. Stop the Sandbox and the Navigator by pressing **Ctrl+C** in the DAML assistant.
```
Comment (tamaskalcza-da) [inline README.md]: Are you sure it's equivalent?

Comment (tamaskalcza-da) [inline README.md]: [Minor] Please, be consistent with the numbering: either use "1" universally or series of numbers. Plugins can help you with that.

Comment (richardkapolnai-da) [reply README.md]: made a note of it on the jira

Comment (richardkapolnai-da) [reply README.md]: I think if md allows multiple ways, and we don't have a strong preference, we are allowed use both. It's almost an extra work to enforce only one.

Comment (richardkapolnai-da) [reply README.md]: They are not equivalent. There is no such thing as `daml stop`. :)

**Pull request merged**